README.md

```diff
@@ -147,3 +147,26 @@ To run the phpunit tests:
 ```bash
 docker-compose exec web ./vendor/bin/phpunit
 ```
+
+## Upgrade from older versions
+
+### Upgrade from 1.0.0-alpha5 and earlier
+
+Target release: You can upgrade directly to the version that contains this README.md.
+
+#### Paragraphs migration
```

Comment:
I don't get this bit, makes more sense to me saying "Upgrading from version x to y" or "Upgrading to version y"